code_challenges.rb

```diff
@@ -1,33 +1,70 @@
 # ASSESSMENT 4: Ruby Coding Practical Questions
 # MINASWAN
 
-# --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
+# --------------------1) Create a method that takes in a number and determines if the number 
+# is even or odd. Use the test variables provided.
 
-num1 = 7
-# Expected output: '7 is odd'
-num2 = 42
-# Expected output: '42 is even'
-num3 = 221
-# Expected output: '221 is odd'
+# num1 = 7
+# # Expected output: '7 is odd'
+# num2 = 42
+# # Expected output: '42 is even'
+# num3 = 221
+# # Expected output: '221 is odd'
 
 
+  if num1.odd? == true
+    puts " #{num1} is very odd "
+  else
+    puts " #{num1} is steady even"
+  
+  end
+  
+  # output =  7 is very odd
+  
+  if num2.odd? == true
+    puts " #{num2} is pretty odd "
+  else
+    puts " #{num2} is adequately even"
 
-# -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
+  end
 
-album1 = 'Rubber Soul'
-# Expected output: 'Rbbr Sl'
-album2 = 'Sgt Pepper'
-# Expected output: 'Sgt Pppr'
-album3 = 'Abbey Road'
-# Expected output: 'bby Rd'
+  # output = 42 is adequately even
+  
+  if num3.odd? == true
+    puts " #{num3} is so odd "
+  else
+    puts " #{num3} is quite even"
+  
+  end
+# output = 221 is so odd
```

Comment:
Good work! See below about wrapping this logic in a method! Please consider refactoring this code to a method: it will help out wrapping your mind around it!